packages/vertica-nodejs/test/integration/client/result-metadata-tests.js

```diff
@@ -7,39 +7,32 @@ new helper.Suite().test('should return insert metadata', function () {
   pool.connect(
     assert.calls(function (err, client, done) {
       assert(!err)
+      client.query(
+        'CREATE TEMP TABLE zugzug(name varchar(10))',
+        assert.calls(function (err, result) {
+          assert(!err)
+          assert.equal(result.oid, null)
+          assert.equal(result.command, 'CREATE')
 
-      helper.versionGTE(
```

Comment:
Why is this removed? I'm not sure what this was originally trying to do.

Reply:
Anywhere you see the versionGTE() call it's testing some split in functionality between two different version of postgres. Does not apply to vertica.